presto-docs/src/main/sphinx/release/release-330.rst

```diff
@@ -0,0 +1,131 @@
+=========================
+Release 330 (XX Feb 2020)
+=========================
+
+General Changes
+---------------
+
+* Fix incorrect behavior of :func:`format` for ``char`` values. Previously, the function
+  did not preserve trailing whitespace of the value being formatted. (:issue:`2629`)
+* Fix query failure in some cases when aggregation uses inputs from both sides of a join. (:issue:`2560`)
+* Fix query failures when dynamic filtering is on for certain multi-level joins. (:issue:`2659`)
+* Fix failure of ``SELECT`` queries when accessing ``information_schema`` schema tables with empty
+  value used in predicate (like ``SELECT * FROM catalog.information_schema.tables WHERE table_name=''``).
+  (:issue:`2575`)
+* Fix query failure when :doc:`/sql/execute` is used with an expression containing a function call. (:issue:`2675`)
+* Fix query failure for certain co-located joins when dynamic filtering is enabled. (:issue:`2685`)
+* Fix failure in ``SHOW CATALOGS`` when the user does not have permissions to see any catalogs. (:issue:`2593`)
+* Improve query performance for some join queries when :doc:`/optimizer/cost-based-optimizations`
+  are turned on. (:issue:`2722`)
+* Prevent uneven distribution of data that could occur when writing data
+  if redistribution or writer scaling is enabled. (:issue:`2788`)
+* Add support for ``CREATE VIEW`` with comment (:issue:`2557`)
+* Add support for all major geometry types to :func:`ST_Points`. (:issue:`2535`)
+* Add ``required_workers_count`` and ``required_workers_max_wait_time`` session properties
+  to control the number of workers that must be present in the cluster before the query
+  starts. (:issue:`2484`)
+* Add ``physical_input_bytes`` column to ``system.runtime.tasks`` table. (:issue:`2803`)
+* Verify that the target schema exists for the :doc:`/sql/use` statement. (:issue:`2764`)
+* Verify that the session catalog exists when executing :doc:`/sql/set-role`. (:issue:`2768`)
+
+
+Server Changes
+--------------
+
+* Require running on Java 11. This requirement may be temporarily relaxed by adding
+  ``-Dpresto-temporarily-allow-java8=true`` to the Presto :ref:`presto_jvm_config`.
+  This fallback will be removed in future versions of Presto after March 2020. (:issue:`2751`)
+* Add experimental support for running on Linux aarch64 (ARM64). (:issue:`2809`)
+
+Security Changes
+----------------
+
+* :ref:`principal-rules` are deprecated and will be removed in a future release.
```

Comment:
@dain, ``:ref:`principal-rules` `` doesn't work. Are we missing a label somewhere?

Reply:
There is ref: ```.. _principal_rules:``` in ```presto-docs/src/main/sphinx/security/built-in-system-access-control.rs```